chapters/modules/chapter_8/pages/solidity_verifier.adoc

```diff
@@ -0,0 +1,37 @@
+[id="verifiers_solidity"]
+
+= Verifiers in Solidity
+
+L1 smart contract that verifies STARK proofs from SHARP.
```

Comment:
In the discussion below, we encounter several contracts:

1. Starkware Sharp Verifier
2. GpsStatementVerifier
3. CpuFrilessVerifier
4. MemoryPageFactRegistry

For the sake of clarity and educational value, could we include a brief description of each contract's function in this context? Understanding the individual purpose and interplay of these contracts would greatly enhance our comprehension of the overall system. For example, it would be helpful to know why the second contract is named 'GpsStatementVerifier'.

Reply:
tbh, I'm not sure what to put on description for CpuFrilessVerifier and MemoryPageFactRegistry

Reply:
LGTM. I completely understand. Thank you @trangnv!